GVFS/GVFS.Service/GVFSService.Mac.cs

```diff
@@ -35,8 +55,19 @@ public void RunWithArgs(string[] args)
 
             try
             {
-                this.serviceThread.Start();
-                this.serviceThread.Join();
+                this.repoRegistry.Upgrade();

```

Comment:
Since we never supported version 1 on Mac there is no need to call `this.repoRegistry.Upgrade();`.

(This doesn't hurt anything, but is a small perf hit as it causes us to re-write out the registry each time).